metaflow/plugins/kfp/kfp.py

```diff
@@ -306,6 +309,14 @@ def build_kfp_component(node: DAGNode, task_id: str) -> KfpComponent:
                     ),
                     None,  # default
                 ),
+                pytorch_decorator=next(
+                    (
+                        deco
+                        for deco in node.decorators
+                        if isinstance(deco, PyTorchDistributedDecorator)
+                    ),
+                    None,  # default
```

Comment:
Happy to leave this in for now but I think it'd be good to develop an abstraction here as I'm guessing this isn't super scalable to inject named decorators like this as we add more of them :)

Reply:
True, trying to find a more extensible modular way would be beneficial, maybe after seeing a few instantiations like this.  These decorators modify or change the KFP Pipeline DAG.